grakn-core/src/main/java/ai/grakn/concept/Relation.java

```diff
@@ -66,9 +67,19 @@
      *
      * @return A list of all the Instances and their Role Types.
      */
+    @Deprecated // TODO: Remove this method
     Map<RoleType, Instance> rolePlayers();
 
     /**
+     * Retrieves a list of every {@link Instance} involved in the {@link Relation}, filtered by {@link RoleType} played.
+     * @param roleTypes used to filter the returned instances only to ones that play any of the role types.
+     *                  If blank, returns all role players.
+     * @return a list of every {@link Instance} involved in the {@link Relation}.
+     */
+    // TODO: Rename this method to `rolePlayers`
+    Collection<Instance> newRolePlayers(RoleType... roleTypes);
```

Comment:
I don't think this is that useful. Without the context of the role it would be weird. Imagine you have ten roles all played by the same instance? 

I would like to remove this.